src/@carbon-info/views/Features/Features.tsx

```diff
@@ -300,50 +322,52 @@ const Features: React.FC = () => {
               today.
             </Typography>
           </div>
-          <Carousel
-            ref={carouselRef}
-            responsive={Responsive.roadmap}
-            containerClass={clsx(classes.carouselContainer, "carousel-container")}
-            arrows={false}
-            infinite={true}
-            showDots
-            dotListClass={classes.dotList}
-            minimumTouchDrag={150}
-            customTransition="opacity 10ms ease-in"
-            transitionDuration={10}
-            itemClass={classes.item}
-            autoPlay={scrolledPastFeatures}
-            autoPlaySpeed={5000}
-            beforeChange={handleBackgroundAnimations}
-          >
-            {items.map((item, index) => {
-              return (
-                <Box className={classes.carouselItem} key={`${item.header}-${index}`}>
-                  <Box>
-                    <Typography variant="h2" color="textPrimary" align="left" style={{ marginBottom: "40px" }}>
-                      {item.header}
-                    </Typography>
-                    <Typography variant="body1" color="textSecondary" align="left" className={classes.description}>
-                      <span style={{ color: theme.palette.text.primary }}>{item.subheader}</span>
-                      &nbsp;{item.description}
-                    </Typography>
-                    <CTAButton
-                      text="Read Our Docs"
-                      link={item.ctaLink}
-                      textClassName={classes.ctaButtonText}
-                      iconClassName={classes.ctaButtonIcon}
+          <Box className={classes.carouselWrapper} onMouseOver={handleFeaturesMouseOver} onMouseLeave={handleFeaturesMouseLeave}>
```

Comment:
1. Since you used `onMouseLeave`, think it's better to use `onMouseEnter` instead.

2. When we access carbon website on tablet and scroll down to Features scroll section, it looks quite off-center for tablet.
  https://github.com/Switcheo/carbon-website/assets/65583522/bdb47ada-1a60-489f-8be0-2a3f92bf56a5